admin/app/components/solidus_admin/ui/tab/component.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::UI::Tab::Component < SolidusAdmin::BaseComponent
+  SIZES = {
+    s: %w[h-7 px-1.5 body-small-bold],
+    m: %w[h-9 px-3 body-small-bold],
+    l: %w[h-12 px-4 body-text-bold],
+  }
+
+  TAG_NAMES = {
+    a: :a,
+    button: :button,
+  }
+
+  def initialize(text:, size: :m, tag: :a, disabled: nil, active: nil, **attributes)
```

Comment:
```suggestion
  def initialize(text:, size: :m, tag: :a, disabled: false, active: false, **attributes)
```

What do you think about :arrow_up: ? They're boolean parameters in spirit :slightly_smiling_face:

Reply:
Done, thanks!